Makefile

```diff
@@ -76,7 +76,7 @@ vendor/github.com/libgit2/git2go/v31/static-build: ## Build libgit2
 	./build-libgit2-static
 
 .PHONY: deb
-deb: ## Build the deb, usually called inside the container
+deb: build ## Build the deb, usually called inside the container
```

Comment:
You don't need to build first.  It will build and vet before it's creates the deb.

Reply:
I revalidated without "build" as pre-step its working as expected, let me revert this change.